src/cmdc_tools/datasets/official/AR/data.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import requests
+
+from ..base import ArcGIS
+
+
+class Arkansas(ArcGIS):
+    ARCGIS_ID = "PwY9ZuZRDiI5nXUB"
+
+    def __init__(self, params=None):
+
+        if params is None:
+            params = {
+                "f": "json",
+                "where": "1=1",
+                "outFields": "*",
+                "returnGeometry": "false",
+            }
+
+        super(Arkansas, self).__init__(params)
+
+    def get(self):
+        url = self.arcgis_query_url(
+            service="ADH_COVID19_State_Case_Metrics", sheet=0, srvid=""
+        )
+        res = requests.get(url, params=self.params)
+
+        df = pd.DataFrame.from_records(
+            [x['attributes'] for x in res.json()["features"]]
+        )
+
+        # TODO: Filter columns
+        keep = df.rename(columns={
```

Comment:
These columns don't seem to get renamed or they don't exist. @tlyon3, could you look into what needs to happen for this method to work?